heliopy/data/util.py

```diff
@@ -26,7 +26,8 @@
 
 def process(dirs, fnames, extension, local_base_dir, remote_base_url,
             download_func, processing_func, starttime, endtime,
-            try_download=True, units=None, processing_kwargs={}):
+            try_download=True, units=None, keys=None,
```

Comment:
The new `units` keyword argument needs a docstring entry.

Reply:
yep! doing it now!